lib/fs-loader.js

```diff
@@ -70,60 +70,64 @@ const loadModule = async (dir, options) => {
 
 // Creates Metaschema object and fills it with schemas
 //   dir <string> | <string[]>
-//   loadOptions <Object>
-//     context
-//     decorators
+//   options <Object>
+//     context <Object>
+//     decorators <Object>
 //     localDecorators <Object>, optional
 //       [type] <Object> decorators specific for this schema type
 //     pathToType <Object>
 //       [ext] <string>
 //     pathToType <Function>
 //       path <string>
 //     Returns: <string>
+//     loadOrder <Function>, optional
+//       filename1 <string>
+//       filename2 <string>
+//     Returns: <number>
 //     preprocessor: <Function>, optional
-//       schemas <Schema[]>
-//     Returns: <Schema[]>
+//       schemas <Object[]>
```

Comment:
`<Schema>` is intentional, there is no need to change it to `<Object>` anywhere.